### PR TITLE
Updating Frequency Analysis page to reflect toolkit 1.6.0

### PR DESCRIPTION
**Pull request by WireBaron**
# Description

The Frequency Analysis page under API Reference->Hyperfunctions contained a broken link and hadn't been updated with all of the new methods included in toolkit 1.6.0.

# Version

Which documentation version does this PR apply to?

- [x] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

# Links

Fixes #[insert issue link, if any]

### CODE REVIEW

File: api/frequency-analysis.md [hunk 1]
```diff
@@ -12,9 +12,12 @@ additional hyperfunctions, you need to install the
 |-|-|-|-|-|
 |Frequency|SpaceSavingAggregate|[`freq_agg`](/hyperfunctions/frequency-analysis/freq_agg/)|❌|✅|
 |Frequency|SpaceSavingAggregate|[`topn_agg`](/hyperfunctions/frequency-analysis/topn_agg/)|❌|✅|
```
Comment (charislam) [inline api/frequency-analysis.md:14]:
```suggestion
|||[`topn_agg`](/hyperfunctions/frequency-analysis/topn_agg/)|❌|✅|
```

File: api/frequency-analysis.md [hunk 2]
```diff
@@ -12,9 +12,12 @@ additional hyperfunctions, you need to install the
...
-|Frequency||[`values`](/hyperfunctions/frequency-analysis/values-freq_agg/)|❌|✅|
-|Frequency||[`topn`](/hyperfunctions/frequency-analysis/topn/)|❌|✅|
+|Frequency|SpaceSavingAggregate|[`into_values`](/hyperfunctions/frequency-analysis/into_values-freq_agg/)|❌|✅|
+|Frequency|SpaceSavingAggregate|[`topn`](/hyperfunctions/frequency-analysis/topn/)|❌|✅|
+|Frequency|SpaceSavingAggregate|[`min_frequency`](/hyperfunctions/frequency-analysis/min_frequency-max_frequency/)|❌|✅|
+|Frequency|SpaceSavingAggregate|[`max_frequency`](/hyperfunctions/frequency-analysis/min_frequency-max_frequency/)|❌|✅|
```
Comment (charislam) [inline api/frequency-analysis.md:18]:
```suggestion
|||[`into_values`](/hyperfunctions/frequency-analysis/into_values-freq_agg/)|❌|✅|
|||[`topn`](/hyperfunctions/frequency-analysis/topn/)|❌|✅|
|||[`min_frequency`](/hyperfunctions/frequency-analysis/min_frequency-max_frequency/)|❌|✅|
|||[`max_frequency`](/hyperfunctions/frequency-analysis/min_frequency-max_frequency/)|❌|✅|
```

File: api/frequency-analysis.md [hunk 3]
```diff
@@ -12,9 +12,12 @@ additional hyperfunctions, you need to install the
...
 |Frequency|StateAgg|[`state_agg`](/hyperfunctions/frequency-analysis/state_agg/)|❌|✅|
-|Frequency||[`duration_in`](/hyperfunctions/frequency-analysis/duration_in/)|❌|✅|
+|Frequency|StateAgg|[`duration_in`](/hyperfunctions/frequency-analysis/duration_in/)|❌|✅|
+|Frequency|StateAgg|[`into_values`](/hyperfunctions/frequency-analysis/into_vals-state-agg/)|❌|✅|
```
Comment (charislam) [inline api/frequency-analysis.md:21]:
```suggestion
|||[`duration_in`](/hyperfunctions/frequency-analysis/duration_in/)|❌|✅|
|||[`into_values`](/hyperfunctions/frequency-analysis/into_vals-state-agg/)|❌|✅|
```

File: api/frequency-analysis.md [hunk 4]
```diff
@@ -12,9 +12,12 @@ additional hyperfunctions, you need to install the
 |-|-|-|-|-|
 |Frequency|SpaceSavingAggregate|[`freq_agg`](/hyperfunctions/frequency-analysis/freq_agg/)|❌|✅|
 |Frequency|SpaceSavingAggregate|[`topn_agg`](/hyperfunctions/frequency-analysis/topn_agg/)|❌|✅|
-|Frequency||[`values`](/hyperfunctions/frequency-analysis/values-freq_agg/)|❌|✅|
-|Frequency||[`topn`](/hyperfunctions/frequency-analysis/topn/)|❌|✅|
+|Frequency|SpaceSavingAggregate|[`into_values`](/hyperfunctions/frequency-analysis/into_values-freq_agg/)|❌|✅|
+|Frequency|SpaceSavingAggregate|[`topn`](/hyperfunctions/frequency-analysis/topn/)|❌|✅|
+|Frequency|SpaceSavingAggregate|[`min_frequency`](/hyperfunctions/frequency-analysis/min_frequency-max_frequency/)|❌|✅|
+|Frequency|SpaceSavingAggregate|[`max_frequency`](/hyperfunctions/frequency-analysis/min_frequency-max_frequency/)|❌|✅|
 |Frequency|StateAgg|[`state_agg`](/hyperfunctions/frequency-analysis/state_agg/)|❌|✅|
```
Comment (charislam) [inline api/frequency-analysis.md:19]:
```suggestion
||StateAgg|[`state_agg`](/hyperfunctions/frequency-analysis/state_agg/)|❌|✅|
```

**Review by charislam — APPROVED** [review]
Non-blocking suggestion: easier to read if we don't fill in all the lines with the repeated 'frequency' and 'aggregate' values

**Review by Loquacity — APPROVED** [review]

**Comment by charislam** [comment]
@WireBaron fair enough! Just a heads up that we are going to be automating table creation very shortly so we don't have to update these by hand every time. I think the new table design does make it clearer that the labels span multiple rows but perhaps you want to have a look: https://github.com/timescale/web-documentation/pull/315

**Pull request merged**